### PR TITLE
Add ApiScan to night pipeline

### PR DESCRIPTION
**Pull request by weshaggard**
Add ApiScan to night pipeline

### CODE REVIEW

File: eng/pipelines/aggregate-reports.yml [hunk 1]
```diff
@@ -35,6 +35,14 @@ stages:
                 -WorkingDirectory '$(Build.ArtifactStagingDirectory)'
                 -NupkgFilesDestination 'nupkgFiles'
 
+          - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
```
Comment (jsquire) [inline eng/pipelines/aggregate-reports.yml:38]: What's this do?

Comment (weshaggard) [reply eng/pipelines/aggregate-reports.yml:38]: See https://github.com/Azure/azure-sdk-tools/issues/7085

Comment (jsquire) [reply eng/pipelines/aggregate-reports.yml:38]: After reading the linked docs over a couple of times, I still have no idea what an HPV is or what this actually does - but the take-away is "we have to do this," so I guess that's enough to sate my curiosity.

**Pull request closed without merging**